third_party/move/tools/move-package/Cargo.toml

```diff
@@ -20,27 +34,9 @@ serde = { workspace = true, features = ["derive"] }
 serde_yaml = { workspace = true }
 sha2 = { workspace = true }
 tempfile = { workspace = true }
+termcolor = { workspace = true }
 toml = { workspace = true }
 walkdir = { workspace = true }
-
-evm-exec-utils = { path = "../../evm/exec-utils", optional = true }
-hex = { workspace = true, optional = true }
-move-abigen = { path = "../../move-prover/move-abigen" }
-
-move-binary-format = { path = "../../move-binary-format" }
-move-bytecode-source-map = { path = "../../move-ir-compiler/move-bytecode-source-map" }
-move-bytecode-utils = { path = "../move-bytecode-utils" }
-move-command-line-common = { path = "../../move-command-line-common" }
-move-compiler = { path = "../../move-compiler" }
-move-compiler-v2 = { path = "../../move-compiler-v2" }
-move-core-types = { path = "../../move-core/types" }
-move-docgen = { path = "../../move-prover/move-docgen" }
-move-model = { path = "../../move-model" }
-move-symbol-pool = { path = "../../move-symbol-pool" }
-
-move-to-yul = { path = "../../evm/move-to-yul", optional = true }
-termcolor = { workspace = true }
-
 whoami = { workspace = true }
 
 [dev-dependencies]
```

Comment:
`move-stdlib` could be made `{ workspace = true }`?

Reply:
Replied above